vm_setup.pl

```diff
@@ -341,7 +354,7 @@ sub random_pass {
     my $pwgen_installed=qx[ yum list installed | grep 'pwgen' ];
     if ($pwgen_installed) { 
         print "\npwgen installed successfully, using it to generate random password\n";
-        $password = `pwgen -Bs 15 1`;
+        $password = `pwgen -Bs 25 1`;
```

Comment:
qx() is used in some other places so to be consistent I suggest either backticks everywhere or qx() everywhere.